portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+
+public class Comment {
+    private String username;
+    private String commentText;
+    private boolean isAdmin;
```

Comment:
"private final" for each of these, as the values are only being set once in the constructor.

Reply:
Changed